pkg/transforms/mqtt.go

```diff
@@ -31,10 +31,42 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 )
 
-// MQTTSender ...
+//Mqtt options config
+type MqttConfig struct {
+	qos    		  byte
+	retain 		  bool
+	autoreconnect bool
+}
+
 type MQTTSender struct {
 	client MQTT.Client
 	topic  string
+	opts   MqttConfig
+}
+
+// default mqttConfig
```

Comment:
Please provide complete descriptions in Godoc format. i.e.   "NewMqttConfig returns a new MqttConfig with default values". Same comment for each on the Set APIs below and the structs above. Yea, looks like this need to be done for the original API. ;-)